tests/main/snap-run-explain/expected

```diff
@@ -0,0 +1,72 @@
+The extra output from snap run --explain is not suitable for parsing.
+
+[snap run]
+
+Invoked as "snap"
+Inferred application details:
+  snap name: test-snapd-sh
+  app name: test-snapd-sh
+  confinement: strict
+  base: core (implicit)
+Will exec through the following tools:
+  - snap-confine (execution environment and sandboxing)
+  - snap-exec (command chain and environment variables)
+Creating user data directories
+  - ~/snap/test-snapd-sh/x1
+  - ~/snap/test-snapd-sh/common
+Creating symbolic link "x1" -> "/root/snap/test-snapd-sh/current"
+Executing snap-confine /snap/core/8250/usr/lib/snapd/snap-confine
+  with arguments: snap.test-snapd-sh.test-snapd-sh /usr/lib/snapd/snap-exec test-snapd-sh -c /bin/true
+  with altered environment
```

Comment:
is this the full environment or just what gets added to it ?  if just what's gets added to it  we probably want something different,  also we want a ':' colon here

Reply:
Those are just the alterations, which effectively are just the additions (we don't remove environment variables).

Reply:
Colon added.